src/components/navbar/SideBarData.js

```diff
@@ -6,7 +6,7 @@ import * as MdIcons from 'react-icons/md';
 export const SideBarData = [
     {
         title: "Assignments",
-        path: "/assignments",
+        path: " ",
```

Comment:
An empty string will always take you to the home page. Delete the path for assignments and test. Also remember to make the link in the test dropdown to take you to the tests and not to tests/html.